internal/cache/svg.go

```diff
@@ -57,3 +59,13 @@ func destroyExpiredSvgs(now time.Time) {
 		svgs.Delete(exp)
 	}
 }
+
+func overriddenName(name string, o fyne.CanvasObject) string {
+	if o != nil { // for overridden themes get the cache key right
```

Comment:
When will `o` ever be `nil`?

Reply:
Yes, plenty of times. If the context object is nil. Like how the tests call GetSvg with a nil context. I suppose it is possible to step through every call to that and ensure that GetSvg and SetSvg are always providing an object for context - but this is a handy shortcut and a safety that doesn't seem superfluous.